lib/rubocop/cop/rspec/repeated_example_group_body.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Check for repeated describe and context block body.
+      #
+      # @example
+      #
+      #    # bad
+      #    describe 'cool feature x' do
+      #      it { cool_predicate }
+      #    end
+      #
+      #    describe 'cool feature y' do
+      #      it { cool_predicate }
+      #    end
+      #
+      #    # good
+      #    describe 'cool feature' do
+      #      it { cool_predicate }
+      #    end
+      #
+      #    describe 'another cool feature' do
+      #      it { another_predicate }
+      #    end
+      #
+      #    # good
+      #    context 'when case x', :tag do
+      #      it { cool_predicate }
+      #    end
+      #
+      #    context 'when case y' do
+      #      it { another_predicate }
```

Comment:
It's not clear how this good example differs from the previous one. Perhaps here the body should be the same, but the tag makes the difference?